sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/CollationBenchmark.scala

```diff
@@ -130,6 +215,9 @@ object CollationBenchmark extends CollationBenchmarkBase {
     benchmarkUTFStringEquals(collationTypes, generateSeqInput(10000L))
     benchmarkUTFStringCompare(collationTypes, generateSeqInput(10000L))
     benchmarkUTFStringHashFunction(collationTypes, generateSeqInput(10000L))
+    benchmarkUTFStringContains(collationTypes, generateSeqInput(10000L))
+    benchmarkUTFStringStartsWith(collationTypes, generateSeqInput(10000L))
+    benchmarkUTFStringEndsWith(collationTypes, generateSeqInput(10000L))
```

Comment:
I would just name those `benchmarkContains `, `benchmarkStartsWith`, `benchmarkEndsWith`, since here we are in the context of CollationBenchmark anyways.